app/static/js/vendor/jquery/jquery.checkboxes-1.2.0.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
```

Comment:
@niranjan94 That library takes care of disabling users from selecting multiple checkboxes pretty neatly. Hence was added,  is it strictly necessary to exclude it?